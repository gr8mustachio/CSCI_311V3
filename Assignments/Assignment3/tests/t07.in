powerSet
3
2 5 4
powerSet
4
1 2 3 4
powerSet
5
12 48 13 27 91
