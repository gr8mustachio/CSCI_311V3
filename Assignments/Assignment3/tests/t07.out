Array A:
2 
5 
2 5 
4 
2 4 
5 4 
2 5 4 
Array A:
1 
2 
1 2 
3 
1 3 
2 3 
1 2 3 
4 
1 4 
2 4 
1 2 4 
3 4 
1 3 4 
2 3 4 
1 2 3 4 
Array A:
12 
48 
12 48 
13 
12 13 
48 13 
12 48 13 
27 
12 27 
48 27 
12 48 27 
13 27 
12 13 27 
48 13 27 
12 48 13 27 
91 
12 91 
48 91 
12 48 91 
13 91 
12 13 91 
48 13 91 
12 48 13 91 
27 91 
12 27 91 
48 27 91 
12 48 27 91 
13 27 91 
12 13 27 91 
48 13 27 91 
12 48 13 27 91 
